.flowconfig

```diff
@@ -18,6 +18,7 @@ flow-typed
 
 module.name_mapper='^/\(.*\)' -> '<PROJECT_ROOT>/app/\1'
 suppress_comment= \\(.\\|\n\\)*\\$FlowStyledIssue
+suppress_comment= \\(.\\|\n\\)*\\$ResolveModuleIssue
```

Comment:
does this suppress anything?

Reply:
It supresses this error I get when importing react-beautiful-dnd: 
Cannot resolve module `react-beautiful-dnd`.Flow(InferError)

Reply:
So you decide to use new package and that's just fine. Then you find out there is an error caused by this package. Such things tend to happen time to time. And then instead of solving the error or using another package you decide to suppress it! Brilliant solution? No!

Reply:
This was not meant as a permanent solution... I'll change this.